olio/service/resources/version_resource.go

```diff
@@ -0,0 +1,48 @@
+package resources
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/thedataguild/faer/service/middleware"
+)
+
+type Version struct {
+	skeletonVersion string `json:"skeletonVersion"`
```

Comment:
the skeleton version should not be exposed in the public contract; the skeleton needs to get the framework version some other way